docs/roadmap.md

```diff
@@ -0,0 +1,57 @@
+---
+id: roadmap
+title: Open Source Readiness 2022 Roadmap
+sidebar_label: OSR 2022 Roadmap
+---
+
+This document contains an inital Open Source Readiness roadmap as contributed in [2022 Community Roadmap #23](https://github.com/finos/open-source-readiness/discussions/23#discussioncomment-2253788).
+
+## High Level Description
+
+At a high level, OSR should be an engine to identify shared, financial services specific, challenges to open source participation from financial institutions.
+
+The SIG should prioritize the challenges and then foster the creation of projects that address them.
+
+## OSR Responsibilities
+
+The following lists the 2022 responsibilities of the Open Source Readiness SIG.
+
+### Prioritizing common areas of challenge for financial institutions in progressing through the OS maturity model
+
+- Through prioritization of a backlog (e.g. github issues) at OSR SIG meetings.
+- Processing output of the [OSMM](https://github.com/finos/osmm) survey responses FINOS receives.
+- Through real-life findings FINOS can feed in (e.g. Member XX cannot use DCOs, Member YY can’t access github, etc).
+
+### Explore and develop regulators role in the Open Source maturity of the industry, in conjunction with the FINOS Innovation Regulation SIG
+
+- Between [this discussion](https://groups.google.com/a/finos.org/g/osr/c/GMMD38RkaMY/m/nYMwTdR3BAAJ) on the list and this [github discussion](https://github.com/finos/open-source-readiness/discussions/9), consolidating a shared list of common regulatory requirements industry is concerned about
+- Then, mapping to existing solutions (or start open sources project to address them).
+
+### Support the definition and evolutionj of a truly industry-wide OSMM
+
+- Provide / solicit ongoing Feedback and validation of OSMM ahead of FINOS OSMM launch
+- Participate and amply FINOS OSMM launch
+- Promote participants and adjacent communities to take the survey
+- Propose training / certifications to strengthen the value proposition of OSMM for all constituents
```

Comment:
@mcleo-d  - do you know of existing training / certification best practice, either in FinOS or LF? Would be interested to see what we're shooting for here.

Reply:
Hi @troups - Please note, I have referenced this comment here https://github.com/finos/open-source-readiness/pull/27#issuecomment-1094865935 as I would like to start using the `trainings` section of the OSR microsite to help answer this question 👍🏻